sites/partners/src/listings/PaperListingForm/UnitsSummaryForm.tsx

```diff
@@ -80,6 +82,18 @@ const UnitsSummaryForm = ({
   })
 
   const unitType = formWatch("unitType")
+  const minOccupancy = formWatch("minOccupancy")
+  const maxOccupancy = formWatch("maxOccupancy") || minOccupancy
+  const sqFeetMin = formWatch("sqFeetMin")
+  const sqFeetMax = formWatch("sqFeetMax") || sqFeetMin
```

Comment:
I don't think this or operation should be done here, you are using an or with a value from a hook

I think it would be better to do this or where they are needed